controllers/forum-controllers.js

```diff
@@ -0,0 +1,9 @@
+const {selectReviews} = require('../models/models')
+
+exports.getReviews = (request, response, next)=>{
+  selectReviews().then((reviewsObj)=>{
+    response.status(200).send(reviewsObj)
```

Comment:
I would be tempted to change the name of this to just **reviews**  rather than **reviewsObj** - remember to change on test side also